handbook/strategy/objectives.md

```diff
@@ -0,0 +1,26 @@
+# Launch Pad Objectives
+
+We want to create a set of clear, measurable objectives for each school year, in rough order of priority. This helps hold us accountable to our goals, ensure that we are improving the club, and learn from our experiences.
+
+Objectives and results for each year are documented in this page. Each school year is denoted by the year of the fall semester, typically in September, and ends around May of the following year. At the end of each school year, we will see how close we got to our goals and assess how we can improve the following year.
+
+Objectives have the following format:
+
+- **Objective: some overarching theme we want to work towards**
+  - Result: some measureable result we want to achieve to make progress towards this objective
+
+## 2020
+
+- **Encourage collaboration and communication between developers and designers**
+  - Every single club member participates in at least 3 discussions on GitHub (issues and pull requests)
+
+- **Mentorship & diversity within the club**
+  - 40% of new members are “beginners”
```

Comment:
This is more of a comment rather than a request for change, and something I've thought about recently. I'm concerned that having a large portion of our team comprised of "beginners" may reduce the quality of their Launch Pad experience. Inevitably, some may struggle with anything from git, setup, general implementation questions. The Launch Pad docs are definitely going to help.

Reply:
That's true - but one of my hopes on this front less in-person time translates to more 1-on-1 support time: hopefully, questions can get raised in DMs where they can get addressed or redirected to resources, instead of having to queue up for a lead's time

Still a risk of having a bad experience unfortunately, but I think another of our goals with this 40% mark is that we should be more careful to being investing in longer-term members. More beginners can also help the diversity of our membership too (ie faculty diversity)